Review of Am. Nat. manuscript 58395
===================================

Summary
-------



Suggestion
----------

Based on technical and conceptual aspects, reject with minor revisions. Based on scope and innovation, the manuscript may fall short of providing enough novel insight; it also partially claims to provide novel insight that would actually not seem that sensational given previous work.



Major points
------------

**--** The motivation of the study is formulated in terms of the concepts of evolutionary interest and ecological genetic conflict. However, the model used by the authors is partially a population-genetic model, and so it would be useful to better explain how evolutionary interest relates to individual fitness. Similarly, it would seem appropriate to more explicitly state how evolutionary interest and fitness are related to the reproductive value. 


**--** The population-genetic literature on multilocus models of divergent selection and gene flow relevant to the main topic of the manuscript is quite evolved, but shallowly represented in the Introduction and Discussion. It would help to be more clear about how the current work connects to previous work, and so be more specific about what is really novel in the former. The importance of recombination (a primary aspect of genetic architecture) for the establishment and maintaineance of locally adaptive polymorphisms in the presence of gene flow has been studied exhaustively in previous analytical () and simulation studies (). *Say what these studies find.*
*Give examples*


**--** There is currently a confusion about the primary contribution of the manuscript. The authors seem to be primarily interested in the effect of genetic architecture (essentially recombination rate) on the relative importance of genetics vs. plasticity. My impression is that this is indeed the main point of the paper. The authors also claim to provide novel insight into the evolution of genomic islands of divergence. Yet, the simulations investigating the evolution of these islands for a fixed recombination map ignore the effect of plasticity. Indeed, the results of these simulations do not seem to convey any new insight compared to the existing literature (see e.g. the simulation studies by Feder, Nosil, Flaxman and coauthors), and so are more of a confirming than ground-breaking nature.

The authors do, however, go beyond previous work by explicitly simulating the evolution of recombination between ecological polymorphisms, and between the latter and modifiers of the reaction norm. This is indeed novel and of great interest. It would seem appropriate to state that previous theoretical work (which, notably, ignored plasticity) has predicted that the recombination rate among ecological polymorphisms should evolve (e.g. Aeschbacher & B端rger 2014). In principle, the evolution of lower recombination rates between ecological polymorphisms in the presence of gene flow could be classified as a mechanism leading to genomic islands of divergence. The existing literature on the evolution of genomic islands takes the recombination landscape as static (yet not necessarily constant along the genome), and is more concerned with the dynamics of establishment and maintenance of ecological polymorphisms (e.g. B端rger & Akerman 2011, Aeschbacher & B端rger 2014, Ackermann & B端rger 2014) and the evolution of other aspects of the genetic architecture than recombination (e.g. the stacking of small mutations, Yeaman & Whitlock 2011). While there does not seem to be a firm conceptual barrier to breaking with this paradigm, I do think that it matters on what time scales we can expect the recombination rate to evolve, relative to the time scale of mutation and selection on ecological polymorphisms. I feel it would be beyond the scope of the current manuscript to require a thorough investigation of this point. However, I suggest that the authors discuss it to some greater extent.

**--** Unhappy with the term 'statistical information'. Is it used in previous work? How does it differ from (gametic) linkage disequilibrium, i.e. from correlations among allele-frequencies arising from gene flow and selection? The current formulation seems unnecessarily fancy.


Minor points
------------

I use the following code of abbreviations to classify my points:

Q: question
C: comment
S: suggested change
R: requested change

